src/client/swagger/unity.yml

```diff
@@ -23,174 +23,197 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/date':
+  '/billing':
     get:
-      operationId: GetBillingDate
+      operationId: GetBilling
       tags:
-        - BillingDate
+        - Billing
       responses:
         '200':
-          description: Billing Date for Account
+          description: Billing information for account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BillingDate'
+                $ref: '#/components/schemas/BillingInfo'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/region':
+  '/marketplace':
     get:
-      operationId: GetRegion
+      operationId: GetMarketplace
       tags:
-        - Region
+        - Marketplace
       responses:
         '200':
-          description: Account region for billing
+          description: Marketplace information for account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Region'
+                $ref: '#/components/schemas/Marketplace'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/account/history':
+  '/billing/start_date':
     get:
-      operationId: GetHistory
+      operationId: GetBillingStartDate
       tags:
-        - History
+        - Billing
       responses:
         '200':
-          description: Account history for billing, rates and usage
+          description: Start of the current billing period for the Account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/History'
+                $ref: '#/components/schemas/BillingDate'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/notify_settings':
+  '/usage/vectors':
     get:
-      operationId: GetBillingNotifySettings
+      operationId: GetUsageVectors
       tags:
-        - BillingNotifySettings
+        - Usage
       responses:
         '200':
-          description: Settings for billing notifications
+          description: Usage vectors for the account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BillingNotifySettings'
+                $ref: '#/components/schemas/UsageVectors'
+
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-    put:
-      operationId: PutBillingContact
+  '/usage/{type}':
```

Comment:
I feel weird about having these at the same level as `/usage/vectors`. Assuming `{type}`s are vectors, I would expect this to be `/usage/vectors/:type`. If that's the case I'd probably make the route `/usage_vectors` instead?